src/http/controllers/device.js

```diff
@@ -19,6 +27,29 @@ const getSingleDevice = async (req, res) => {
   }
 };
 
+/**
+ * @description Controller for "update admin user" API operation
+ * @param {object} req Express request object
+ * @param {object} res Express response object
+ */
+const getAllDevice = async (req, res) => {
+  const { user, userType, log } = res.locals;
+  log.debug('Executing the getAllDevice controller');
+  try {
+    const device = await deviceService.getAllDevice(user, userType, log);
```

Comment:
You are fetching multiple devices here so it should be `devices` not `device`.

Reply:
fix